front/src/modules/ui/table/components/EntityTableHeader.tsx

```diff
@@ -55,43 +52,46 @@ export function EntityTableHeader({ viewFields }: OwnProps) {
     {},
   );
   const [columnWidths, setColumnWidths] = useState(initialColumnWidths);
+  const [isResizing, setIsResizing] = useState(false);
+  const [initialPointerPositionX, setInitialPointerPositionX] = useState<
+    number | null
+  >(null);
 
+  const [resizedFieldId, setResizedFieldId] = useState<string | null>(null);
   const [offset, setOffset] = useState(0);
-  const resizedColumnId = useRef('');
-  const initialHandlerPosition = useRef(-1);
```

Comment:
As discussed with @magrinj and @emilienchvt I've created an issue so we can discuss about `useRef` usage: #1009